main.go

```diff
@@ -12,7 +12,10 @@ import(
 
 // sourcePathFile is the file in which the checked out source path
 // will be written to
-const sourcePathFile = "sourcePath"
+const sourcePathFile string= "sourcePath"
```

Comment:
can this path be configured based on `git-repo` version ? ie git-repo >= 2.4 use this path and older ones use the previous path.